dkron/job.go

```diff
@@ -275,6 +275,25 @@ func (j *Job) GetParent() (*Job, error) {
 	return parentJob, nil
 }
 
+// Return the time.Location based on the job's Timezone, or
+// the default (UTC) if none is configured
+func (j *Job) GetTimeLocation() *time.Location {
+	loc, err := time.LoadLocation(j.Timezone)
+	if err != nil {
+		return DefaultJobTimeLocation()
+	}
+	return loc
+}
+
+// Returns a UTC time.Location
+func DefaultJobTimeLocation() *time.Location {
```

Comment:
I would move this to the more general `utils.go` and refactor its name to `DefaultTimeLocation` as it applies to several entities across the code, what do you think?

Reply:
`DefaultJobTimeLocation` is always used alongside the job name. The pair is the job context for an execution. That is why I put it in `job.go`.
If job would have been in its own package, I would have named it `DefaultTimeLocation`, and it could have been more appropriately been referred to as `job.DefaultTimeLocation`.

What it is doing is indeed very generic, though. When reasoned about from the function outwards, disregarding what it is supposed to mean, and only looking at what it does, it would indeed best be placed in `utils.go`. But then I suggest renaming it to `UTCTimeLocation` or `GetUTCTimeLocation`.

I can eliminate two uses of `DefaultJobTimeLocation` by passing a job from further down the callstack (eliminates use in `agent.go` and `store.go`). The remaining occurrences in `job.go` and `dashboard.go` can do without function's "job" semantic and it could then happily be moved to `utils.go`.

Shall I give that a try?

Reply:
I removed DefaultJobTimeLocation altogether.